test/e2e/framework/test_context.go

```diff
@@ -347,6 +349,7 @@ func RegisterClusterFlags(flags *flag.FlagSet) {
 	flags.StringVar(&cloudConfig.ConfigFile, "cloud-config-file", "", "Cloud config file.  Only required if provider is azure or vsphere.")
 	flags.IntVar(&TestContext.MinStartupPods, "minStartupPods", 0, "The number of pods which we need to see in 'Running' state with a 'Ready' condition of true, before we try running tests. This is useful in any cluster which needs some base pod-based services running before it can be used.")
 	flags.DurationVar(&TestContext.SystemPodsStartupTimeout, "system-pods-startup-timeout", 10*time.Minute, "Timeout for waiting for all system pods to be running before starting tests.")
+	flags.IntVar(&TestContext.SystemPodsStartupTimeoutMultiplier, "system-pods-startup-timeout-multiplier", 1, "Timeout multiplier for waiting for all system pods to be running before starting tests.")
```

Comment:
Since this is impacting only external storage tests, https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/testsuites/testdriver.go#L160 may be a better place to add a new parameter.

Reply:
I'd prefer to have this option for in-tree plugins too, Azure tests tend to time out pretty often.

Reply:
testdriver.go is used for in-tree plugins too

Reply:
Looking again at TestDriver struct - if the test timeout multiplier is part of it, then it's much harder to pass the timeouts all over the tests. But it's probably the right thing to do, as each driver can have its own timeout, not affecting the other volume tests.

Reply:
@jsafrane  so you are suggesting putting this timemultiplier inside TestDriver? If that is the case, I'll close this PR for now and investigate it.